### PR TITLE
Removing the 2017 facia election container

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

Attempt two: https://github.com/guardian/frontend/pull/17098

I have done a better job at checking this now 😓 

---

Reverting https://github.com/guardian/frontend/pull/16847, https://github.com/guardian/frontend/pull/16822 and  https://github.com/guardian/frontend/pull/16806

The election container was created just for the election which has now passed. Removing the code. https://trello.com/c/AUdDVElQ/64-election-cleanup

_Note:_ There has been talk of elements of this container to be reimplemented in the future. Until that is decided though its best to revert this change to avoid dead code lying around.

**TODO:**
- [x] Changes were made to the facia tool/config tool that also have to be reverted: https://github.com/guardian/facia-tool/pull/242

cc @stephanfowler @guardian/dotcom-platform

## What is the value of this and can you measure success?
Less code! Less container options! 

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Screenshots
N/A

## Tested in CODE?
Nope

### CODE REVIEW

File: common/app/views/fragments/containers/facia_cards/standardContainer.scala.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
          data-id="@containerDefinition.dataId">
 
         @for(sliceWithCards <- containerLayout.slices) {
-            @slice(sliceWithCards, containerDefinition.index, isElectionContainer = containerDefinition.displayName.contains("general election 2017"), frontProperties = Some(frontProperties))
```
Comment (NataliaLKB) [inline common/app/views/fragments/containers/facia_cards/standardContainer.scala.html]: I had forgotten to remove a `)` here which broke things. This has now been properly removed.

File: common/app/layout/Front.scala [hunk 2]
```diff
@@ -259,7 +259,6 @@ case class FaciaContainer(
 
   def isStoryPackage: Boolean = container match {
     case Dynamic(DynamicPackage) => true
-    case Dynamic(DynamicElection) => true // #election2017
     case _ => false
   }
```
Comment (jfsoul) [inline common/app/layout/Front.scala]: probably not your code but this pattern match is just `container == Dynamic(DynamicPackage)` now?

**Review by jfsoul — APPROVED** [review]
lgtm

File: common/app/layout/Front.scala (hunk 2 above)
Comment (NataliaLKB) [inline common/app/layout/Front.scala]: Yep that was there before - most likely because there used to be other things. Happy to change it though

Comment (jfsoul) [inline common/app/layout/Front.scala]: Maybe don't change it now, especially if already tested without.  It does look like something that might be added to again anyway.

Comment (NataliaLKB) [inline common/app/layout/Front.scala]: Very true.

**Pull request merged**